.github/workflows/ci.yml

```
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - main
+
+env:
+    python_version: 3.6
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ env.python_version }}
+    - uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('setup.py', 'Makefile') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Test
+      run: |
+        python3.6 -m venv .venv
+        source .venv/bin/activate
+        make travis_install
+        # Hack: The use of chrgp compensates for a quirk of Docker. The PyCharm image
+        # used by make format sets up a user called `developer` and assigns it UID
+        # 1000. Travis is running as UID 2000. An alternative would be to pass --user
+        # to `docker run` and bind-mount an /etc/passwd that maps that to `developer`.
+        # Currently I'm unclear why `make format` works unchanged on Docker
```

```suggestion
        # Hack: The use of `chrgp` compensates for a quirk of Docker. The PyCharm image
        # used by `make format` sets up a user called `developer` and assigns it UID
        # 1000. Travis is running as UID 2000. An alternative would be to pass --user
        # to `docker run` and bind-mount an /etc/passwd that maps that to `developer`.
        # Currently, we're unclear why `make format` works as is on developer machines.
```

The reference to Travis is stale. And we should check if the claim for Travis still applies to GH Action, and if this hack is even necessary for GH Actions.

---

It seems like the claim is still valid for GH Actions, removing the hack [caused issues](https://github.com/DataBiosphere/hca-metadata-api/runs/4973363065?check_suite_focus=true#step:5:252).